packages/fxa-auth-db-mysql/scripts/readSecurityEvents.js

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env node
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * This script takes the uid of a user and will print a security event object of that user
+ * from various tables like securityEvents, sessionTokens and devices tables from fxa database
+ */
+
+// Usage: node readSecurityEvents.js uid
+
+const log = {
+  trace: console.log,
+  error: console.error,
+  stat: console.log,
+  info: console.log
+} //eslint-disable-line no-console
+const DB = require('../lib/db/mysql')(log, require('../db-server').errors)
+const config = require('../config')
+
+const securityEventsObject = {}
+
+// This input of user's data can be modified according to the usecase
+const user = {
+  // email: 'farhan@gmail.com',
+  // uid: '2A4A31EBAC9E46D9AD483B92789EC934',
+  uid: process.argv[2],
+  ipAddr: '127.0.0.1'
```

Comment:
Ah, now I see why this is needed, the [fetchSecurityEvents_1](https://github.com/mozilla/fxa/blob/bd8ff27e8d9ec32b55bc43dfb8e179e752990bba/packages/fxa-auth-db-mysql/lib/db/schema/patch-032-033.sql#L74) stored procedure requires it.